src/services/localStorage.tsx

```diff
@@ -0,0 +1,38 @@
+import {Dispatch} from "react";
+import {TeamType} from "./getTeamTokenMock.tsx";
+
+export type StorageType = {
```

Comment:
I think this is the contract that you want to follow, like a `Port` so maybe it could be just `Storage` or `TeamStorage`, and your module below could be called `LocalStorage` could be called `TeamLocalStorage` because it's the adapter.